prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -0,0 +1,365 @@
+import data_helper
+import const
+import dwelling
```

Comment:
This will need to be modified to refer to `prereise.gather.demanddata.transportation_electrification.*`